pkg/cmd/crdconversionwebhook/converter/framework.go

```diff
@@ -78,13 +78,13 @@ func serve(w http.ResponseWriter, r *http.Request, convert convertFunc) {
 	contentType := r.Header.Get("Content-Type")
 	serializer := getInputSerializer(contentType)
 	if serializer == nil {
-		msg := fmt.Sprintf("invalid Content-Type header `%s`", contentType)
+		msg := fmt.Sprintf("invalid Content-Type header `%q`", contentType)
```

Comment:
i thought `%q` added quotes around the string, do we also want the  single quotes  as well ?

eg something like this
```suggestion
		msg := fmt.Sprintf("invalid Content-Type header %q", contentType)
```